app/models/order.rb

```diff
@@ -0,0 +1,5 @@
+class Order < ApplicationRecord
+	has_many   :order_items
+	belongs_to :user
+	validates  :customer_id, :total_price, presence: true
```

Comment:
where is customer_id in order ?

Reply:
no need for total price validation, we can add a default value of total price to 0

Reply:
add validation to discount should be within 0 to 100 range

Reply:
how belongs_to :user will work with customer_id

Reply:
tells to user the foreign key of order is customer_id